### PR TITLE
Dart SDK is no longer required, we predownload it

### LINKED ISSUE #1886: Improve the Flutter SDK archive page
Page URL: https://flutter.io/docs/development/tools/sdk/archive.html
Page source: https://github.com/flutter/website/tree/master/src/docs/development/tools/sdk/archive.md

-  [ ] **Add a description of each of the channels.** As requested by @Hixie (and reported by @sfshaza2): the page should contain a description of the channels.

    The channels are well documented in the [Flutter's channels][] wiki page, which is referenced at the start of the Archive page:

    > The Stable channel contains the most stable Flutter builds. See [Flutter’s channels][] for details.

    Is this good enough? If not, then I'd rather not have duplicate information. I'd then suggest moving at least the initial part of the wiki page text into the Archive page. WDYT @Hixie?

    Also, you mentioned that you might prefer an interface similar to the [Dart SDK Archive][] page: with **Version** and **OS** drop downs? Or were you only giving that page as an example w.r.t. the channel descriptions?

- [ ] list sha256sums for download files (originally requested by @cbracken in #1955)

- [x] **Question** is the **Ref** value really useful for users? @cbracken @Hixie

Overall, maybe we should just carry over the [Dart SDK Archive][] design and implementation? Thoughts?

cc @kwalrath 

[Dart SDK Archive]: https://www.dartlang.org/tools/sdk/archive
[Flutter's channels]: https://github.com/flutter/flutter/wiki/Flutter-build-release-channels#flutters-channels

### PR DESCRIPTION
**Pull request by Hixie**
Also fixes #1886, #1885, #1884.

Now with one less trailing space.

### CODE REVIEW

**Comment by Hixie** [comment]
This is the same as https://github.com/flutter/website/pull/162 with has LGTM @abarth.

This time I won't immediately check it in as a pavlovian reaction to getting an LGTM.

**Comment by abarth** [comment]
LGTM

(This is a test.)

File: getting-started.md [hunk 1]
```diff
@@ -6,42 +6,22 @@ permalink: /getting-started/
 ---
 
 Flutter currently supports developers on Mac and Linux (64-bit).
-Windows support is planned.
+Windows support is planned but currently incomplete.
```
Comment (eseidelGoogle) [inline getting-started.md]: https://github.com/flutter/flutter/issues/138 if you want it.

File: getting-started.md [hunk 2]
```diff
@@ -6,42 +6,22 @@ permalink: /getting-started/
...
-is not found, you will need to add `DIRECTORY_WHERE_DART_SDK_IS_LOCATED/bin`
-to your PATH.
-
-For example:
-
-```
-$ export PATH=<DART_SDK_DOWNLOAD_LOCATION>/bin:$PATH
-```
+* bash, mkdir, rm
+* git
```
Comment (eseidelGoogle) [inline getting-started.md]: The first time you type 'git' on mac, it shows a dialog about the command-line developer tools.  Unclear if we need to walk people through this?

Comment (eseidelGoogle) [inline getting-started.md]: screen shot 2016-02-16 at 10 48 00 am

File: getting-started.md [hunk 3]
```diff
@@ -55,12 +35,11 @@ the `flutter` command.)
...
 use our scripts and tools.
 
-```
-$ cd <directory where you cloned the flutter repo>
-$ export PATH=`pwd`/bin:$PATH
-```
+`export PATH=` _directory where you cloned the flutter repo_ `/bin:$PATH`
 
-Run `flutter --version` to ensure the `flutter` command is on your PATH.
+Open a new shell and run `flutter --version` to ensure the `flutter` command is on your PATH.
```
Comment (eseidelGoogle) [inline getting-started.md]:
```
flutter --version
Updating flutter tool...
/Users/ericseidel/flutter/bin/flutter: line 21: pub: command not found
```

Comment (eseidelGoogle) [inline getting-started.md]: I guess I'm not supposed to be on alpha?

File: getting-started.md [hunk 4]
```diff
@@ -55,12 +35,11 @@ the `flutter` command.)
...
+The first time you do this will take a few seconds as Flutter will first download the Dart SDK then precompile itself.
+Subsequent runs should be much faster.
```
Comment (eseidelGoogle) [inline getting-started.md]: Flutter will only need to do these steps after subsequent `flutter upgrade` commands?

**Pull request merged**